Sources/BraintreeCore/Public/BraintreeCore/BTAppContextSwitcher.h

```diff
@@ -0,0 +1,85 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+#pragma mark - BTAppContextSwitchDriver protocol
+
+/**
+ A protocol for handling the return from gathering payment information from a browser or another app.
+ @note The app context may switch to a SFSafariViewController or to a native app, such as Venmo.
+*/
+@protocol BTAppContextSwitchDriver
```

Comment:
We went with "driver" rather than "handler" in this name since the classes that implement this protocol are both drivers: `BTVenmoDriver` and `BTPaymentFlowDriver`.